xsnippet/api/conf.py

```diff
@@ -12,23 +12,24 @@
 import os
 import configparser
 
+import decouple
 
-def get_conf(paths, envvar=None):
+
+def get_conf(envvar=None):
     """Return one settings instance combines from different sources.
 
     The idea that lies behind that function is to gather config settings
     from different sources, including dynamic once pointed by passed
     environment variable (usually production overrides).
 
-    :param paths: a list of paths to configuration files
-    :type paths: [str]
-
     :param envvar: an environment variable that points to additional config
     :type envvar: str
 
-    :return: a configuration instance
-    :rtype: :class:`configparser.ConfigParser`
+    :return: a configuration dictionary
+    :rtype: :class:`dict`
     """
+    # TODO: Drop INI support and use configuration via environment vars only.
```

Comment:
Please, notice this. :) The code of this function is terrible but it's a temporary measure to maintain backward compatibility for a while.